lib/tasks/import_schools.rake

```diff
@@ -1,9 +1,24 @@
 namespace :import do
   desc 'Import schools from GIAS'
   task schools: :environment do
+    puts 'Import schools from GIAS'
     StageGiasDataJob.perform_now
   end
 
+  desc 'Process schools records from DataStage::School to School'
+  task process_schools_data: :environment do
+    service = SchoolUpdateService.new
+
+    # TODO: Can be removed once we have data for these RB
+    DataStage::School.gias_status_open.where('responsible_body_name like ?', '%Northampton%').destroy_all
+    DataStage::School.gias_status_open.where('responsible_body_name like ?', '%Overseas Establishments%').destroy_all
```

Comment:
@tonyheadford this is what I had to do so that I can successfully create the other schools.